testsuite/features/secondary/minkvm_guests.feature

```diff
@@ -342,6 +342,95 @@ Feature: Be able to manage KVM virtual machines via the GUI
     Then I wait until I do not see "test-net1" text
     And I should not see a "test-net1" virtual network on "kvm_server"
 
+# Start provisioning scenarios
+
+# This scenario needs the 15-SP2 base system channel to be synchronized
+@long_test
+@virthost_kvm
+@scc_credentials
+  Scenario: Create auto installation distribution
+    Given I am on the Systems page
+    When I follow the left menu "Systems > Autoinstallation > Distributions"
+    And I follow "Create Distribution"
+    And I enter "SLE-15-SP2-TFTP" as "label"
+    And I enter "/usr/share/tftpboot-installation/SLE-15-SP2-x86_64" as "basepath"
+    And I select "SLE-Product-SLES15-SP2-Pool for x86_64" from "channelid"
+    And I select "SUSE Linux Enterprise 15" from "installtype"
+    And I enter "useonlinerepo insecure=1" as "kernelopts"
+    And I click on "Create Autoinstallable Distribution"
+    Then I should see a "Autoinstallable Distributions" text
+    And I should see a "SLE-15-SP2-TFTP" link
+
+@long_test
+@virthost_kvm
+@scc_credentials
+  Scenario: Create auto installation profile
+    Given I am on the Systems page
+    And I follow the left menu "Systems > Autoinstallation > Profiles"
+    And I follow "Upload Kickstart/Autoyast File"
+    When I enter "15-sp2-kvm" as "kickstartLabel"
+    And I select "SLE-15-SP2-TFTP" from "kstreeId"
+    And I select "KVM Virtualized Guest" from "virtualizationTypeLabel"
+    And I attach the file "/sle-15-sp2-autoyast.xml" to "fileUpload"
+    And I click on "Create"
+    Then I should see a "Autoinstallation: 15-sp2-kvm" text
+    And I should see a "Autoinstallation Details" text
+    When I enter "self_update=0" as "kernel_options"
+    And I click on "Update"
+    And I follow "Variables"
+    And I enter "distrotree=SLE-15-SP2-TFTP\nregistration_key=1-SUSE-PKG-x86_64" as "variables" text area
+    And I click on "Update Variables"
+    And I follow "Autoinstallation File"
+    Then I should see a "SLE-15-SP2-TFTP" text
+
+@long_test
+@virthost_kvm
+@scc_credentials
+  Scenario: Create an auto installing KVM virtual machine
+    Given I am on the "Virtualization" page of this "kvm_server"
+    When I follow "Create Guest"
+    And I wait until I see "General" text
+    And I enter "test-vm2" as "name"
+    And I select "15-sp2-kvm" from "cobbler_profile"
+    And I click on "Create"
+    Then I should see a "Hosted Virtual Systems" text
+    When I wait until I see "test-vm2" text
+    And I wait until table row for "test-vm2" contains button "Stop"
+    # Test the VM boot params
```

Comment:
Instead of a comment, if we really want to advise about what it's doing here, I would prefer a new Scenario, with a title of "Test the VM boot params"

Reply:
yeah right, but that doesn't tell the reader what we expect